src/c++/perf_analyzer/docs/README.md

```diff
@@ -48,3 +48,6 @@ model input data, the performance measurement modes, the performance metrics and
 outputs, how to benchmark different servers, and more.
 
 - [Perf Analyzer CLI](cli.md)
+- [Data Guide](data_guide.md)
+- [Measurements & Metrics](measurements_metrics.md)
+- [Benchmarking](benchmarking.md)
```

Comment:
Are you combining more than one JIRA issue in this PR?

Reply:
yes. 3 in total